### PR TITLE
[DCSFUS-2] Develop a Product Page using HTML

### PR DESCRIPTION
**Pull request by Nimi-1**
Screenshot (7)
Screenshot (8)

### CODE REVIEW

**Review by Mayowa2020 — APPROVED** [review]
Well done @Nimi-1 ! Kindly run the test on your code and revert. Please tag me on Slack @Bukky      once you are done, or you are experiencing issues running the test.

File: submissions/toluwanimiObafemi/html/product.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8"> 
+    <title>Tolu's Product page</title>
+</head>
+
+<body>
+    <p> <a href="product.html"> Visit the HP store</a></p>
```
Comment (ToluEnny24) [inline submissions/toluwanimiObafemi/html/product.html:10]: Here a href should point to # not product.html

File: submissions/toluwanimiObafemi/html/product.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+            <hr>
+            <p>
+                HP 15-f272wm
+            </p>
+            <p>
+              price:  $349.00
+            </p>
+        </section>
+        <section>
+            <p> <a href="product.html"> Add to Cart</a></p>
```
Comment (ToluEnny24) [inline submissions/toluwanimiObafemi/html/product.html:30]: Here a href should point to # not product.html

File: submissions/toluwanimiObafemi/html/product.html [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        <section>
+            <article>
+                <p> <b>Note:</b> Products with electrical plugs are designed for use in the US. Outlets and Voltage differ internationally and this product may require an adapter or converter for use in your destination: Please check compatibility before purchasing</p>
+            </article>
+        </section>
+
+        <section>
+            <h3> You may also like</h3>
+            <img src="Images/HP 15-3.jpg" width="100" height="80">
+            <a href="product.html"> 2022 Newest HP 15 15.6" HD Display</a>
```
Comment (ToluEnny24) [inline submissions/toluwanimiObafemi/html/product.html:99]: Here the tag a href should point to # not product.html

File: submissions/toluwanimiObafemi/html/product.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+            <img src="Images/HP 15-4.jpg" width="100" height="80">
+            <a href="product.html"> 2021 Flagship Pavillon 15.6" HD Display</a>
```
Comment (ToluEnny24) [inline submissions/toluwanimiObafemi/html/product.html:101]: Here the tag a href should point to # not product.html

File: submissions/toluwanimiObafemi/html/product.html [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+            <img src="Images/HP 15-5.jpg" width="100" height="80">
+            <a href="product.html"> HP 255 G8 Notebook, 15.6" HD Display</a>
+        </section>
+
+        <section>
+            <p> <a href="product.html"> Home</a></p>
```
Comment (ToluEnny24) [inline submissions/toluwanimiObafemi/html/product.html:107]: Here a href should point to # not product.html

File: submissions/toluwanimiObafemi/html/product.html [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+        </section>
+
+        <section>
+            <h3> You may also like</h3>
+            <img src="Images/HP 15-3.jpg" width="100" height="80">
+            <a href="product.html"> 2022 Newest HP 15 15.6" HD Display</a>
+            <img src="Images/HP 15-4.jpg" width="100" height="80">
+            <a href="product.html"> 2021 Flagship Pavillon 15.6" HD Display</a>
+            <img src="Images/HP 15-5.jpg" width="100" height="80">
+            <a href="product.html"> HP 255 G8 Notebook, 15.6" HD Display</a>
```
Comment (ToluEnny24) [inline submissions/toluwanimiObafemi/html/product.html:103]: Here a href should point to # not product.html

File: submissions/toluwanimiObafemi/html/product.html (hunk 3 above)
Comment (ToluEnny24) [inline submissions/toluwanimiObafemi/html/product.html:99]: REmove all href tags pointing to product .html and replace them with #

**Review by ToluEnny24 — CHANGES_REQUESTED** [review]
Hi @Nimi-1 well done and your task has stayed longer than it should. Remove all hrefs tags pointing to product.html and replace with #. Also the success criteria states that you should us two images. Remove the other images and stick to two. You can use as many images for your 
 individual test work locally but not on this repo. If you need help you can reach out too

**Pull request merged**